### ISSUE TITLE
Cypress config env variable when defining get Property '<var_name>' does not exist on type 'Partial<PluginEnvOptions>'

### ISSUE THREAD

**Issue by mluk-sastrify** [filiphric/cypress-plugin-api#107]
I'm using Cypress with Typescript, but when I am defining the variable with `Cypress.env()` command I am getting the following error in VSCode:
![image](https://user-images.githubusercontent.com/115544064/214357297-243a3166-876d-47a8-8d8d-110f705dee0a.png)

`cypress.config.ts` content:
```js
import { defineConfig } from 'cypress';

export default defineConfig({
  e2e: {
    baseUrl: 'http://localhost:4040',
    env: {
      apiServerUrl: 'http://localhost:7777/api',
  },
  setupNodeEvents(on, config) {
    // implement node event listeners here
  },
}
});
```

spec example: `cypress/e2e/example.cy.ts` (as per Cypress docs: https://docs.cypress.io/api/cypress-api/env#No-Arguments):
```js
const { apiServerUrl } = Cypress.env();

describe('GET', () => {

  it('should GET all subscriptions', () => {
    cy.api({
      method: 'GET',
      url: `${apiServerUrl}/todo`,
    }).then(({ body, status }) => {
      expect(status).to.eq(200);
      expect(body).to.have.length(1);
    });
  });
});
```

If I add a empty string in params the error goes away:
![image](https://user-images.githubusercontent.com/115544064/214357540-8808e2fd-8a5e-43c4-a6d2-931f9d6f577c.png)

Seems to be related to: https://github.com/filiphric/cypress-plugin-api/issues/56

**Comment by filiphric** [comment]
Thanks for the report! I just released the fix with version 2.10.1

**Comment by mluk-sastrify** [comment]
Thanks, this is fixed now :)
